src/mspdb.cpp

```diff
@@ -83,24 +90,31 @@ bool tryLoadMsPdb(const char* version, const char* mspdb, const char* path = 0)
 
 bool tryLoadMsPdbVS2017(const char* mspdb, const char* path = 0)
 {
-	const char* key = "SOFTWARE\\Microsoft\\VisualStudio\\SxS\\VS7";
-
-	HKEY hkey;
-	if (RegOpenKeyExA(HKEY_LOCAL_MACHINE, key, 0, KEY_OPEN_FLAGS, &hkey) != ERROR_SUCCESS)
+	ISetupConfigurationPtr query;

```

Comment:
I think the registry key version can stay as another option to find VS2017 (after the COM version), so better create a new function.